src/FluentResults/Results/Result.cs

```diff
@@ -107,7 +108,34 @@ public override string ToString()
         {
             return Result.Ok(value);
         }
-
+        
+        /// <summary>
+        /// Deconstruct Result
+        /// </summary>
+        /// <param name="isSuccess"></param>
+        /// <param name="isFailed"></param>
+        /// <param name="value"></param>
+        public void Deconstruct(out bool isSuccess, out bool isFailed, out TValue value)
```

Comment:
I would challenge the isFailed parameter here. Is it really necessary to return isSuccess and IsFailed. Maybe isSuccess is enough? I think that the consumer of the result object doesn't need both variables. What do you think?

Reply:
i agree. isSuccess is Enough in most case but i think isFailed is also can be handy 
by the way as i refer in playground consumers just can Ignore isFailed by using `_`
` var (isSuccess2, _, value2) = Result.Ok(500);`

its not so necessary but i think having all major properties in Deconstruct fit better 

by the way its not challenging. i will use your opinion